sample-compose/app/src/main/java/co/nimblehq/sample/compose/extensions/SavedStateHandleExt.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.sample.compose.extensions
+
+import androidx.lifecycle.SavedStateHandle
+
+fun <T> SavedStateHandle.getAndRemove(key: String): T? {
```

Comment:
```suggestion
fun <T> SavedStateHandle.getThenRemove(key: String): T? {
```
I think we better use `then` word.

Reply:
fixed in https://github.com/nimblehq/android-templates/pull/530/commits/4ed2969da6824ce98cefed38d01dedb7c7a5d788